website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
```

Comment:
Let's name the anonymous function here so we have some chance of remembering what it does when we revisit it in a year. :-)